src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -43,17 +43,22 @@ StatusNotifierItem::StatusNotifierItem(QString id, QObject *parent)
     mTitle("Test"),
     mStatus("Active"),
     mMenu(nullptr),
-    mMenuExporter(nullptr)
+    mMenuExporter(nullptr),
+    mSessionBus(QDBusConnection::connectToBus(QDBusConnection::SessionBus, mService))
 {
+    // we created a separate DBus connection, because multiple dbus services with _different_
+    // behavior are not supported on QDbus
```

Comment:
Commit ammended. Changed the comment as You suggested and also the commit messge.